Jenkinsfile

```diff
@@ -1100,6 +1108,10 @@ def loadConfigEnvVars(){
   generatorPatterns.addAll(getVendorPatterns('generator/common/beatgen'))
   generatorPatterns.addAll(getVendorPatterns('metricbeat/beater'))
   env.BUILD_GENERATOR = isChangedOSSCode(generatorPatterns)
+
+  // Skip all the stages for PRs with changes in the docs only
+  env.ONLY_DOCS = !params.runAllStages &&
+                  isGitRegionMatch(patterns: [ '^docs/.*' ], shouldMatchAll: true)
```

Comment:
Will this catch changes to files with paths such as `./auditbeat/build/docs/shared/settings60.asciidoc`?

Reply:
Additionally there are also "source" doc files such as `./filebeat/docs/modules/apache.asciidoc`. So I'm wondering if this check needs be more about catching any files that end in `.asciidoc`?

Reply:
That's indeed a good point, I actually thought it was just this initial PR, and then I found I missed the `make update`, so working on supporting those particular scenarios too. Thanks for the heads up :) I just moved the PR to draft to work on it.

Regarding the files, I see `png` files too, I don't know if there are some other type of files. If no, then I can include it in the filter to be file based rather than folder based as you suggested

Reply:
Good point re: `png` files. And, as you said, there might be others too. 

How about we start with `asciidoc` for now, since those do change relatively often, compared to other "non-code" types. And maybe we can add more types later?

Reply:
png is also in the list of allowed files, let's then plan the following support files, if any, with some follow-ups. :)